packages/usa-file-input/src/index.js

```diff
@@ -350,6 +350,22 @@ const addPreviewHeading = (fileInputEl, fileNames) => {
   fileInputEl.setAttribute("aria-label", changeItemText);
 };
 
+/** Add an error listener to the image preview to set a fallback image
+ * @param {HTMLImageElement} previewImage - The image element
+ * @param {String} fallbackClass - The CSS class of the fallback image
+ */
+const setPreviewFallback = (previewImage, fallbackClass) => {
+  previewImage.addEventListener(
+    "error",
+    () => {
```

Comment:
**issue**:  File input event listeners are added in `init()`, can we listen for the error there?